src/wkb.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * WKB (Well-Known Binary) decoder for geometry objects.
+ *
+ * @import {DataReader, Geometry} from '../src/types.js'
+ * @param {DataReader} reader
+ * @returns {Geometry} geometry object
+ */
+export function wkbToGeojson(reader) {
+  const flags = getFlags(reader)
+
+  if (flags.type === 1) { // Point
+    return { type: 'Point', coordinates: readPosition(reader, flags) }
+  }
+
+  if (flags.type === 2) { // LineString
+    return { type: 'LineString', coordinates: readLine(reader, flags) }
+  } else if (flags.type === 3) { // Polygon
```

Comment:
nit: the coding style changes here.

if - return
if - return

but now:

else if - return
else if - return